org.gustavoperedo.FontDownloader.json

```diff
@@ -0,0 +1,60 @@
+{
+    "app-id" : "org.gustavoperedo.FontDownloader",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.38",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "fontdownloader",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=home"
```

Comment:
What does it need home access for?

Reply:
Currently it needs home access to automatically install the fonts to the .fonts folder, but in the future I'm planning on adding a GTKFileChooser dialog for the user to choose where to download the fonts to.

Reply:
You shouldn't use .fonts but rather .local/share/fonts. And you can request just access to that instead of the whole filesystem.

Reply:
Out of curiosity, is there any difference between .fonts and local/share/fonts? Also, can I do this for the next release or should I change for this one?

Reply:
The .fonts directory is deprecated and shouldn't be used

Reply:
I'll update the code, close this request and open a new one, but first I need to know how do I add two directories to the --filesystem ?

Reply:
By adding the filesystem finish arg twice

Reply:
Thank you very much.